authors/apps/authentication/views.py

```diff
@@ -2,15 +2,15 @@
 from rest_framework.generics import RetrieveUpdateAPIView
 from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.response import Response
-from rest_framework.views import APIView
+from rest_framework.generics import CreateAPIView
```

Comment:
@denomash Any reason for us to switch to generic views?

Reply:
Swagger best works with generics, unlike views.

Reply:
Noted.